src/chrome/content/rules/Tutorialspoint.xml

```diff
@@ -0,0 +1,15 @@
+<ruleset name="Tutorialspoint">
+
+	<!--	^tutorialspoint.com/ do not exist.	-->
```

Comment:
Secure connection refused on `dev.`

Please move these comments on the very top of the file.